db-client-java/src/main/java/com/eventstore/dbclient/PersistentSubscriptionConfig.java

```diff
@@ -0,0 +1,130 @@
+package com.eventstore.dbclient;
+
+public class PersistentSubscriptionConfig {
+    private boolean resolveLinktos;
+    private long startFrom;
+    private long messageTimeoutMilliseconds;
+    private boolean extraStatistics;
+    private long maxRetryCount;
+    private long liveBufferSize;
+    private long bufferSize;
+    private long readBatchSize;
+    private boolean preferRoundRobin;
+    private long checkPointAfterMilliseconds;
+    private long minCheckPointCount;
+    private long maxCheckPointCount;
+    private long maxSubscriberCount;
+    private ConsumerStrategy namedConsumerStrategy;
+
+    public void setResolveLinktos(boolean resolveLinktos) {
+        this.resolveLinktos = resolveLinktos;
+    }
+
+    public void setStartFrom(long startFrom) {
+        this.startFrom = startFrom;
+    }
+
+    public void setMessageTimeoutMilliseconds(long messageTimeoutMilliseconds) {
```

Comment:
Maybe an extra `setMessageTimeout(Duration)` would be nice 😄😬 
Same for `setCheckPointAfterMilliseconds`

Reply:
It's not meant to be used by the user. We should only have getters here. A relic of when the JSON parsing used reflection.

Reply:
It's not meant to be used by the user. These setters are there just for the JSON parsing. `PersistentSubscriptionConfig` is not used as a parameter by the client. It's only used to read a persistent subscription current configuration.